### ISSUE TITLE
[Woo POS] VoiceOver Improvements

### ISSUE THREAD

**Issue by staskus** [woocommerce/woocommerce-ios#14957]
Tested POS with VoiceOver and Screen Curtain Turned On (Three Finger Triple Tap).

- [ ] When we enter variable products or check out, the first thing that is read is the "Back button". However, with native navigation bars, the first thing that is read is the content of the page.
- [ ] Two-finger scrub (move two fingers back and forth three times quickly, making a “z”) action is used to go back or dismiss an alert. However, it doesn't work on POS to come back from Checkout or from Variable products.
- [ ] There's no indication that the item was successfully added to the cart. Just a simple "Added" response when Voice Over is on could be great.

**Comment by dangermattic** [comment]
🚫 Please add a type label (e.g. **type: enhancement**) and a feature label (e.g. **feature: stats**) to this issue.

**Comment by toupper** [comment]
Closing as not planned. This is an old, stale POS issue and we're consolidating active Point of Sale tracking in Linear. If this still needs to be addressed, please comment or reopen and we'll bring it back into the backlog.
